records_mover/db/redshift/redshift_db_driver.py

```diff
@@ -59,6 +62,7 @@ def temporary_s3_directory_loc(self) -> Iterator[BaseDirectoryUrl]:
             with self.s3_temp_base_loc.temporary_directory() as temp_loc:
                 yield temp_loc
 
+    # TODO: this should be in loader class
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker